datasets/nz/russia_sanctions/nz_russia_sanctions.yml

```diff
@@ -42,6 +42,7 @@ lookups:
       - match:
           - unknown
           - 08/15/1977
+          - 45625
```

Comment:
fun fact: this is an excel date

```
 h.convert_excel_date(45625)
'2024-11-29T00:00:00'
```

Reply:
I suspected it, but couldn't find a way to verify it :)